src/assets/styles/baseStyles.scss

```diff
@@ -13,8 +13,14 @@ html{
   scroll-behavior: smooth;
 }
 
+
 body{
   font-family: 'Helvetica', sans-serif;
   font-size: 1.6rem;
-  margin: 0;
+  background-image: url('../images/background-light.png');
```

Comment:
it would be better to switch the styles in js instead of changing it manually in css.
in the future changing style will be connected with clicking the button, so js has to have easy access to both themes.

you can try creating a style that can be applied to other element (body is not visible withing index.js) or maybe creating image element that will be appended to index.js